src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinInetProxyHelper.cs

```diff
@@ -79,6 +82,25 @@ public bool ManualSettingsOnly
 
         public string ProxyBypass { get; set; }
 
+        public bool RecentAutoDetectionFailure
+        {
+            get
+            {
+                if (!_autoDetectionFailed)
+                {
+                    return false;
+                }
+                else if ((DateTime.Now - _lastTimeAutoDetectionFailed) <= _recentAutoDetectionInterval)
+                {
+                    return true;
+                }
+                else
+                {
+                    return false;
+                }
```

Comment:
Couldn't the else just return the result of the comparison rather than branching to return true or false?

Reply:
Fixed.